fearless/Modules/Staking/Stories/Model/StoriesFactory.swift

```diff
@@ -0,0 +1,54 @@
+import Foundation
+
+protocol StoriesFactoryProtocol {
+    static func createModel() -> StoriesModel
+}
+
+final class StoriesFactory { }
+
+extension StoriesFactory: StoriesFactoryProtocol {
+    static func createModel() -> StoriesModel {
+
+        // swiftlint:disable line_length
+        let slides1 = [
+            Slide(description: "Staking refers to the process of a Proof-of-Stake (PoS) token-holder locking a tokens in order to participate in the upkeep of the PoS network (e.g. validating transactions; composing new blocks). This increases the security and reliability of the network and is an alternative to mining in Proof-of-Work systems (e.g. Bitcoin). Token-holders who participate in staking are compensated through block rewards and transaction fees.",
```

Comment:
pass current localization, use R.string..

Reply:
I think, we are not going to localize it any time soon, so let's just leave it as it is. When we have finalized version of texts, I'll do it as the way it should be